### ISSUE TITLE
Default template Blazor Server App razor page design mode issues

### ISSUE THREAD

**Issue by vinoopillai** [radzenhq/radzen-blazor-studio#8]
Hello Team,
Congrats on the first CTP release of radzen blazor studio. I could not resist to try it out right away. I created a default dot net 6 server application and opened it. I was able to run the application with out any issues. However opening the counter.razor page crashes the application displaying a black screen. I have attached the log.txt.

[log.txt](https://github.com/radzenhq/radzen-blazor-studio/files/8373533/log.txt)

After that I tried to open the sample northwind blazor  application here the run button is disabled and opening razor pages shows error in design and output window.

I am attaching the log and output windows grab FYI

[log.txt](https://github.com/radzenhq/radzen-blazor-studio/files/8373597/log.txt)
[output.txt](https://github.com/radzenhq/radzen-blazor-studio/files/8373624/output.txt)
Do let me know if you need any further details.

**Comment by pius-faessler** [comment]
FYI: I have the same errors in the log.file with a new .NET 6 WASM app I made.

**Comment by akorchev** [comment]
This could be one of the issues that we fixed with the update that was just released - 1.0.0.preview.2. Please try updating.

**Comment by pius-faessler** [comment]
The update fixed my errors. Thank you very much!

**Comment by vinoopillai** [comment]
To resolve first issue
1. Update to latest preview
2. uninstall the vs 2022 17.2 preview 2 which I had on my PC [was causing conflicts with radzen]
this resolved my issues with design mode of default template application

To resolve second issue
I had to scaffold the northwind application again and could get it working

Then I tried opening my existing radzen app which did not work. But could get it working after removing nodejs installed on my pc [not sure if it is effects radzen studio]

So my issues are resolved now.
Thanks
